myapp/spec/system/tasks_spec.rb

```diff
@@ -27,6 +27,32 @@
           expect(page).to have_content task_2.description
         end
       end
+
+      context 'when sort tasks' do
+        let!(:task_1) { create(:task, title: 'Task 1') }
+        let!(:task_2) { create(:task, title: 'Task 2', created_at: DateTime.now + 1) }
+        let(:sorted_tasks_ids) { page.all('.task-id').map(&:text).map(&:to_i) }
+
+        context 'by ID' do
```

Comment:
maybe you can add the test case for default order!

Reply:
sure, that's better~